dot/core/service.go

```diff
@@ -43,6 +43,8 @@ import (
 
 var _ services.Service = &Service{}
 
+var maxResponseSize = 8 // maximum number of block datas to reply with in a BlockResponse message.
```

Comment:
is this configurable ?

Reply:
not right now, I picked it based on what I saw to work and not work. we could change it in the future if needed